src/hotspot/share/runtime/vframeArray.cpp

```diff
@@ -612,8 +613,7 @@ void vframeArray::unpack_to_stack(frame &unpack_frame, int exec_mode, int caller
       Bytecode_invoke inv(caller, elem->bci());
       // invokedynamic instructions don't have a class but obviously don't have a MemberName appendix.
       // NOTE:  Use machinery here that avoids resolving of any kind.
-      const bool has_member_arg =
-          !inv.is_invokedynamic() && MethodHandles::has_member_arg(inv.klass(), inv.name());
+      const bool has_member_arg = inv.has_member_arg();
```

Comment:
I reckon the comment about invokedynamic isn't needed anymore. It could be moved to has_member_arg if you want to keep it.

Reply:
Good idea.  Done!